clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus_test.go

```diff
@@ -46,8 +46,16 @@ var (
 func turnOffLoggingToStderrInKlog() {
 	if klogLogToStderr {
 		klog.InitFlags(nil)
-		flag.Set("logtostderr", "false")
-		flag.Set("v", "2")
+		err := flag.Set("logtostderr", "false")
+		if err != nil {
+			klog.Errorf("Unable to set flag %v", err)
```

Comment:
Instead of this, let's:
- pass `*t.Testing` as a parameter to this function
- use t.Errorf(...) here (and below)